### ISSUE TITLE
How to remove "artificial accepted usage for a synonym"?

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#2490]
**How to remove &quot;artificial accepted usage for a synonym&quot;?**

"Carex pendula" was originally published in the "Global Register of Introduced and Invasive Species - Belgium", as an accepted name for "Carex agastachys": `https://www.gbif.org/species/162644337.` We have since (March 19) reverted back to the name "Carex agastachys" (keeping the same taxonID), since "Carex pendula" is the name of another native species to Belgium

On the backbone page for  Carex pendula however, the Belgian register is still listed, with a reference to `https://www.gbif.org/species/159747346,` which states is a:

> Artificial accepted usage for a synonym if its missing to preserve the taxonomic hierarchy.

Is there a way to remove this, so the Belgian register is no longer listed as a source for Carex pendula being introduced in Belgium?

-----

Github user: @peterdesmet
User: [See in registry](https://www.gbif.org/api/feedback/user/c0921bb5f30b2f42b40e1481de5198eb:1144fe8dd625a6d78b2405953e3ff2d775cf068f2cf8fa1596155fb190aa6a62f6efb4aec71578166703e84fd95c13b3a41b11f009ea47817d2faa0570d06920)
System: Chrome 80.0.3987 / Mac OS X 10.14.6
Referer: https://www.gbif.org/species/159747346
Window size: width 1368 - height 766
[API log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2020-03-20T15:12:13.848Z',mode:absolute,to:'2020-03-20T15:18:13.848Z'))&_a=(columns:!(_source),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
[Site log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2020-03-20T15:12:13.848Z',mode:absolute,to:'2020-03-20T15:18:13.848Z'))&_a=(columns:!(_source),index:'prod-portal-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
System health at time of feedback: INFO

**Comment by ManonGros** [comment]
Hi @peterdesmet
Let me know if this issue has gone out of scope.

Right now the [Global Register of Introduced and Invasive Species - Belgium checklist](https://www.gbif.org/dataset/6d9e952f-948c-4483-9807-575348147c7e) published on GBIF shows that Carex agastachys L.f. is a synonym of Carex pendula Huds.:
<img width="617" alt="Screenshot 2021-09-02 at 11 28 09" src="https://user-images.githubusercontent.com/7677271/131819558-32088501-218e-49ee-bdef-0ce9edd75269.png">

The checklist doesn't have any entry for Carex pendula Huds. This is why Carex pendula Huds. is now labelled as "doubtful" in its GRIIS Belgium checklist version on GBIF https://www.gbif.org/species/159747346

You could consider updating the checklist to show Carex agastachys L.f. as an accepted name.

However, it wouldn't change the GBIF backbone. The current source for both species, the [World Checklist of Selected Plant Families](https://www.gbif.org/dataset/92562ba8-21af-4836-a80a-c31237efe3c5) via the CoL, shows Carex agastachys L.f. as a synonym of Carex pendula Huds: https://www.gbif.org/species/2725229
Do you think this is incorrect? Let us know and we will forward the issue.

**Comment by ManonGros** [comment]
@peterdesmet 
> 1.  Will the record for Carex agastachys L.f. be treated differently if we set the taxonomicStatus to ACCEPTED in our source checklist? E.g. will https://www.gbif.org/species/159747346/verbatim no longer be created?

Yes, changing taxonomicStatus to ACCEPTED and replacing the name and ID of Carex pendula Huds. by  Carex agastachys L.f.'s in the acceptedNameUSage and acceptedNameUsageID fields should make the entry disappear.

Thanks we will forward the issue about Carex agastachys L.f. to the CoL repository.

**Comment by peterdesmet** [comment]
I have updated the source checklist, removing `taxonomicStatus`, `acceptedNameUsage` and `acceptedNameUsageID` (https://github.com/trias-project/unified-checklist/commit/93b901caf1839b25ae5b190e4eb9881102d43c9c), since we consider all our taxa as their own concepts, i.e. accepted.

Once the crawling is finished, https://www.gbif.org/species/162644337/verbatim should no longer list those fields and https://www.gbif.org/species/159747346/verbatim should disappear.

**Comment by peterdesmet** [comment]
This issue is now resolved!
